src/server/tiered_storage.cc

```diff
@@ -590,7 +589,6 @@ PrimeValue TieredStorage::Warmup(DbIndex dbid, PrimeValue::CoolItem item) {
 
   // Bring it back to the PrimeTable.
   DCHECK(hot.ObjType() == OBJ_STRING);
-  hot.SetTouched(true);
```

Comment:
Not sure I understand why 🤔 So we're okay with having it offloaded immediately

Reply:
it was a mistake, the attribute is attached to a key, not value and it's been set on a key in db_slice.cc when it's been looked up. so it's gonna be done automatically.